week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -0,0 +1,103 @@
+// The below functions are syntactically correct but not outputting the right results.
+// Look at the tests and see how you can fix them.
+
+function mood() { //creating a function called mood
+    let isHappy = true; //assigning boolean variables either true or false
+    let isHappy = false;
```

Comment:
Have you tried running this code to see if it will work? I don't think this will compile